Booting/linux-bootstrap-1.md

```diff
@@ -117,45 +117,45 @@ db 0x55
 db 0xaa
 ```
 
-Build and run this with:
+Compile e execute o código acima com:
 
 ```
 nasm -f bin boot.nasm && qemu-system-x86_64 boot
 ```
 
-This will instruct [QEMU](http://qemu.org) to use the `boot` binary that we just built as a disk image. Since the binary generated by the assembly code above fulfills the requirements of the boot sector (the origin is set to `0x7c00` and we end with the magic sequence), QEMU will treat the binary as the master boot record (MBR) of a disk image.
+Isso irá instruir o [QEMU](http://qemu.org) a usar o binário `boot` que compilamos acima como uma imagem de disco. Como o binário gerado pelo código assembly satisfaz os requerimentos para um setor de boot (o início é setado para `0x7c00` e terminado com a sequência mágica) o QEMU tratará o executável como um master boot record (MBR) de uma imagem de disco.
 
-You will see:
+Você verá o seguinte:
 
-![Simple bootloader which prints only `!`](http://oi60.tinypic.com/2qbwup0.jpg)
+![Um bootloader simples que imprime `!`](http://oi60.tinypic.com/2qbwup0.jpg)
 
-In this example we can see that the code will be executed in 16 bit real mode and will start at `0x7c00` in memory. After starting, it calls the [0x10](http://www.ctyme.com/intr/rb-0106.htm) interrupt, which just prints the `!` symbol; it fills the remaining 510 bytes with zeros and finishes with the two magic bytes `0xaa` and `0x55`.
+Nesse exemplo podemos ver que o código será executado em modo real de 16 bits iniciando no endereço em memória`0x7c00`. Depois de inicializar, ele chama o interruptor [0x10](http://www.ctyme.com/intr/rb-0106.htm) que somente imprime o símbolo `!`. Ele também preenche os 510 bytes restantes com zeros e finaliza com os dois bytes mágicos `0xaa` e `0x55`.
 
-You can see a binary dump of this using the `objdump` utility:
+Você pode ver um dump do binário utilizando a ferramenta `objdump`:
 
 ```
 nasm -f bin boot.nasm
 objdump -D -b binary -mi386 -Maddr16,data16,intel boot
 ```
 
-A real-world boot sector has code for continuing the boot process and a partition table instead of a bunch of 0's and an exclamation mark :) From this point onwards, the BIOS hands over control to the bootloader.
+Um setor de boot real teria mais código para seguir com o processo de boot e uma tabela de partição ao invés de vários zeros e um ponto de exclamação :) Daqui por diante a BIOS não tem mais controle sob o bootloader.
```

Comment:
The translation of `hands over control to the bootloader` is like `transfere o controle para o bootloader` instead of `não tem mais controle sob o bootloader`